src/migrations/20201130144748-create-grantee.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
     await queryInterface.createTable('Grantees', {
       id: {
         allowNull: false,
-        autoIncrement: true,
+        autoIncrement: false,
```

Comment:
Why do we need to turn the `auto-incing` off? Are we using an HSES id as our primary key?

Reply:
Yes, the idea is to use HSES ids as the primary keys, since in production grantees will only come from HSES, and use seeders to prepopulate in non-prod environments.